src/plugins/WebOfTrust/util/DelayedBackgroundJob.java

```diff
@@ -0,0 +1,26 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * A {@link BackgroundJob} that is executed after triggers have been aggregated for some delay.
+ *
+ * @author bertm
+ */
+public interface DelayedBackgroundJob extends BackgroundJob {
+    /**
+     * Triggers the execution of the job after either aggregating triggers for at most some delay
+     * specified by the implementation or as soon as the currently running job finishes, whichever
+     * comes last. Implementations must ensure that this method is safe to invoke from any thread
+     * at any moment.
+     */
+    @Override
+    public void triggerExecution();
```

Comment:
(Notice this: Of course the new sentence "Triggers execution of the job after aggregating triggers for at least the given delay." is not correct by itself because another call to triggerExecution() can shorten the delay. **BUT** the next sentence says "If called multiple times, the job will only be executed once, and the shortest given delay will be used." which makes it correct again because it mentions the shorter delay. So using the first sentence together with the second allows us to use the "at least" wording to stress the general intention behind this stuff)